user_guides/jakartaee/src/main/jbake/content/commonappdeploy.adoc

```diff
@@ -1,7 +1,7 @@
 type=page
 status=published
 title=Common Applications Deployment
-next=csiv2logs.html
+next=jaspic-files.html
 prev=portingpackage.html
```

Comment:
Since we are removing csiv2logs.html, we need to update the prev + next entries in for the docs that were previous/next to the CSIV2 doc (kind of like removing a node from a doubly linked list :-).